### DISCUSSION TITLE
Nuova versione del registro

### DISCUSSION THREAD

**Discussion by giadaschiavone** [iisgiua/giuaschool#505] [Idee e proposte]
Buongiorno! Vorrei sapere che funzioni avrà la nuova versione del registro. Magari, potrebbe essere implementata la funzione che anche l'amministratore del registro può creare e inserire circolari e avvisi. Chissà! Fatemi sapere!

**Comment by Lucaccc** [comment]
> Buongiorno! Vorrei sapere che funzioni avrà la nuova versione del registro. Magari, potrebbe essere implementata la funzione che anche l'amministratore del registro può creare e inserire circolari e avvisi. Chissà! Fatemi sapere!

Perché dovrebbe essere abilitato a farlo? Di norma è la/il dirigente l'unico membro autorizzato a sottoscrivere circolari e avvisi.

**Reply by giadaschiavone** [reply]
Ciao Lucacc! Bhè, non proprio. Anche i membri dello staff sono autorizzati, non solo il dirigente.

**Reply by Lucaccc** [reply]
> Ciao Lucacc! Bhè, non proprio. Anche i membri dello staff sono autorizzati, non solo il dirigente.

Ciao!
Non credo sia così: le circolari rappresentano un atto amministrativo ufficiale emanato solo dal Dirigente scolastico, in quanto devono essere firmate da un superiore gerarchico all'interno della scuola (ovvero lo stesso dirigente).
Non a caso, l’art.4, comma 2, del d.lgs.165/2001, cita: "Ai dirigenti spetta l'adozione degli atti e provvedimenti amministrativi, compresi tutti gli atti che impegnano l'amministrazione verso l'esterno, nonché la gestione finanziaria, tecnica e amministrativa mediante autonomi poteri di spesa di organizzazione delle risorse umane, strumentali e di controllo. Essi sono responsabili in via esclusiva dell'attività amministrativa, della gestione e dei relativi risultati." e le circolari sono di carattere amministrativo.
Quanto agli avvisi credo ci sia meno rigidità, ma non per le circolari.
Ad ogni modo è già previsto dal registro l'inserimento di avvisi e circolari da parte di docenti con ruolo di Staff.
